### PR TITLE
feat: added dynamic feeAmount parameter to fungible and non-fungible token create with custom fee method

### LINKED ISSUE #329: In TokenCreateCustom contract, the IHederaTokenService.FixedFee can be set to accept dynamic params
### Problem

Currently in createFungibleTokenWithCustomFeesPublic and createNonFungibleTokenWithCustomFeesPublic, the fixed fee is constructed in a way that accepts 1 for the amount as default.
```
fixedFees[0] = IHederaTokenService.FixedFee(1, fixedFeeTokenAddress, false, false, treasury);
```
This can be updated so that IHederaTokenService.FixedFee can accepts dynamic values from the parameters.

### Solution

Add an `uint256 feeAmount` param to the parameter list then replaced 

```
fixedFees[0] = IHederaTokenService.FixedFee(1, fixedFeeTokenAddress, false, false, treasury);
```

with 
```
fixedFees[0] = IHederaTokenService.FixedFee(feeAmount, fixedFeeTokenAddress, false, false, treasury);
```

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by quiet-node**
**Description**:
- Added dynamic feeAmount parameter to fungible and non-fungible token create with custom fee method for TokenCreateCustom contract which is served in the System Contract DApp
- Fixed unit tests to adapt new feeAmount parameter
- updated System Contract DApp UI to adapt new feeAmount parameter

**Related issue(s)**:

Fixes #329 

**UI updates**:
- new fee amount field is added to FT and NFT token create tabs

Screenshot 2024-03-06 at 12 16 03 PM

Screenshot 2024-03-06 at 12 18 49 PM

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: system-contract-dapp-playground/src/api/hedera/hts-interactions/tokenCreateCustom-interactions/index.ts [hunk 1]
```diff
@@ -73,7 +75,8 @@ export const createHederaFungibleToken = async (
   treasury: string,
   inputKeys: ICommonKeyObject[],
   msgValue: string,
-  feeTokenAddress?: string
+  feeTokenAddress?: string,
+  feeTokenAmount?: number
```
Comment (acuarica) [inline system-contract-dapp-playground/src/api/hedera/hts-interactions/tokenCreateCustom-interactions/index.ts:79]: small nit, should we use `feeAmount` to be consistent with the rest of the PR?

**Review by acuarica — DISMISSED** [review]
lgtm

File: system-contract-dapp-playground/src/components/common/methods/handleSanitizeFormInputs.tsx [hunk 2]
```diff
@@ -133,6 +135,8 @@ export const handleSanitizeHederaFormInputs = ({
       sanitizeErr = 'Invalid denomination token ID';
     } else if (!isAddress(treasury)) {
       sanitizeErr = 'Invalid treasury account address';
+    } else if (withCustomFee && Number(feeTokenAmount) <= 0) {
+      sanitizeErr = 'Custom fee amount must be positive';
```
[the remark this answers is not in the document]
Comment (quiet-node) [reply system-contract-dapp-playground/src/components/common/methods/handleSanitizeFormInputs.tsx:139]: Nice input! Though I think we can address it in another PR.

File: system-contract-dapp-playground/src/api/hedera/hts-interactions/tokenCreateCustom-interactions/index.ts (hunk 1 above)
Comment (quiet-node) [reply system-contract-dapp-playground/src/api/hedera/hts-interactions/tokenCreateCustom-interactions/index.ts:79]: Gotcha I don't know why I left it as feeTokenAmount! Good catch thanks!

**Review by acuarica — APPROVED** [review]
lgtm

**Pull request merged**